workshop/content/docs/publishing/ogcapi-maps.md

```diff
@@ -5,3 +5,104 @@ title: Exercise 5 - Maps of geospatial data via OGC API - Maps
 # Exercise 5 - Maps of geospatial data via OGC API - Maps
 
 TODO
+
+
+[OGC API - Maps](https://ogcapi.ogc.org/maps) provides a Web API to access
+any geospatial data as a georefenced map image.
+
+* [OGC API - Maps](https://docs.ogc.org/DRAFTS/20-058.html) (**draft**)
+
+## pygeoapi support
+
+pygeoapi supports the OGC API - Maps draft specification, using [MapServer MapScript](https://www.mapserver.org/mapscript) and a WMS facade as core backends.
+
+!!! note
+
+    See [the official documentation](https://docs.pygeoapi.io/en/latest/data-publishing/ogcapi-maps.html) for more information on supported map backends
+
+## Serve a WMS via OGC API - Maps
+
+Let's use pygeoapi's WMSFacade provider as a bride to serve an OGC WMS via OGC API - Maps.
+
+We can use the MapServer demo server at https://demo.mapserver.org/cgi-bin/msautotest
+
+!!! note
+
+    Feel free to use the WMS of your choice as you wish!
+
+!!! question "Update the pygeoapi configuration"
+
+    Open the pygeoapi configuration file in a text editor. Add a new dataset section as follows:
```

Comment:
See Features section: thought we decided to use uncommenting. So something like
*...scroll to the line "## START - EXERCISE 5 - Maps", uncomment or paste the configuration snippet below until the line that reads "## END - EXERCISE 5 - Maps". Be sure to keep the proper YAML indentation.*